kie-server-parent/kie-server-wars/kie-server/src/main/webc-resources/META-INF/context.xml

```diff
@@ -6,9 +6,4 @@
             factory="org.jboss.narayana.tomcat.jta.TransactionManagerFactory"/>
   <Resource name="TransactionSynchronizationRegistry" type="javax.transaction.TransactionSynchronizationRegistry"
             factory="org.jboss.narayana.tomcat.jta.TransactionSynchronizationRegistryFactory"/>
-
-
-  <Resource name="jdbc/jbpm" auth="Container" type="javax.sql.DataSource"
```

Comment:
I believe this now requires update to the readme file as it describes how to set this up...

Reply:
+1, will do once we tomcat 9.0.13 is out and this PR is updated accordingly